homeassistant/components/zha/core/channels/general.py

```diff
@@ -360,6 +359,13 @@ def __init__(self, cluster: zigpy.zcl.Cluster, ch_pool: ChannelPool) -> None:
             )
             self.ZCL_INIT_ATTRS["backlight_mode"] = True
             self.ZCL_INIT_ATTRS["power_on_state"] = True
+        elif self.cluster.endpoint.model == "TS011F":
```

Comment:
Why not this instead ?
```python
        if self.cluster.endpoint.model in (
            "TS011F",
            "TS0121",
            "TS0001",
            "TS0002",
            "TS0003",
            "TS0004",
        ):
            self.ZCL_INIT_ATTRS = (  # pylint: disable=invalid-name
                self.ZCL_INIT_ATTRS.copy()
            )
            self.ZCL_INIT_ATTRS["backlight_mode"] = True
            self.ZCL_INIT_ATTRS["power_on_state"] = True
            if self.cluster.endpoint.model == "TS011F":
                self.ZCL_INIT_ATTRS["child_lock"] = True
```
I think it's the fanciest way (@TheJulianJES https://github.com/home-assistant/core/pull/83233#discussion_r1041409678 😉).

Reply:
Pretty elegant.